<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="OutText">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>151</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ProcessButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ShowProcessCheck">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show process</string>
     </property>
    </widget>
    <widget class="QWidget" name="RulesWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>311</height>
      </rect>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Operator:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Output:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="RulleInput">
         <item>
          <widget class="QLineEdit" name="RuleInputEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="RuleOperatorEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="RuleOutputEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="Rules">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>221</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Rules:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddRuleButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add rule</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="InputStringLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Input string:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AlphButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set alphabet</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AlphInput">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="InputStringText">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="InputStringLabel_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Alphabet:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteRuleButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete rule</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
